godtools/ViewControllers/Tract/TractViewControllerContent.swift

```diff
@@ -106,6 +97,11 @@ extension TractViewController {
         }
         
         self.pagesViews.removeAll()
+        resetPagesView()
+    }
+    
+    func resetPagesView() {
+        self.pagesViews = [BaseTractView?](repeating: nil, count: 10)
```

Comment:
Oh! that is a horrible mistake, sorry! was so focus on something I have on paper that forgot to use the variable